src/extensions/default/InlineBoxShadowEditor/BoxShadowEditor.js

```diff
@@ -0,0 +1,268 @@
+define(function(require, exports, module) {
```

Comment:
Can you fix all new files to use 4-space indents please?  There are some other indent issues below as well.

Reply:
tinycolor-min is used in the InlineColorEditor at a stage for detecting if the color value is valid. 
I didn't get the complete hang of it's but I used it anyway.